### PR TITLE
Add database size metric

### PR DESCRIPTION
**Pull request by CharlesJUDITH**
This change will add the database size (db_size) metric.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -296,6 +296,14 @@ TBD:
 	hash_index_cells_used
 	additional_pool_alloc
 
+### Database size
+
+Collected by parsing the output of `SELECT table_schema 'db_name',  Round(Sum(data_length + index_length) / 1024 / 1024, 1) 'db_size_mb'  FROM   information_schema.tables WHERE table_schema not in ('mysql', 'information_schema', 'performance_schema', 'heartbeat') GROUP  BY table_schema;`:
```
Comment (pdumontcriteo) [inline README.md]: Why round(xxx, 1) instead of round(xxx, 0)?

File: mysql.py [hunk 2]
```diff
@@ -402,6 +402,14 @@ def fetch_mysql_slave_stats(conn):
 	status['slave_stopped'] = 1 if slave_row['Slave_SQL_Running'] != 'Yes' or slave_row['Slave_IO_Running'] != 'Yes' else 0
 	return status
 
+def fetch_mysql_db_size(conn):
+	result = mysql_query(conn, "SELECT table_schema 'db_name',  Round(Sum(data_length + index_length) / 1024 / 1024, 1) 'db_size_mb'  FROM   information_schema.tables WHERE table_schema not in ('mysql', 'information_schema', 'performance_schema', 'heartbeat') GROUP  BY table_schema;")
```
Comment (pdumontcriteo) [inline mysql.py]: round(xxx, 0) instead then now conversion needed later

File: mysql.py [hunk 3]
```diff
@@ -402,6 +402,14 @@ def fetch_mysql_slave_stats(conn):
...
+
+	stats = {}
+	for row in result.fetchall():
+    		stats[row['db_name']] = int(row['db_size_mb'])
```
Comment (pdumontcriteo) [inline mysql.py]: Maybe get rid of int conversion?

**Review by pdumontcriteo — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (CharlesJUDITH) [reply README.md]: Indeed round(xxx,0 is better).

File: mysql.py (hunk 3 above)
Comment (CharlesJUDITH) [reply mysql.py]: Yep with round(xxx, 0), int is useless.

File: mysql.py (hunk 2 above)
Comment (CharlesJUDITH) [reply mysql.py]: done

**Review by pdumontcriteo — APPROVED** [review]

**Comment by pdumontcriteo** [comment]
Will expose the size of each user databases

**Pull request merged**